specs/music_spec.rb

```diff
@@ -0,0 +1,45 @@
+require_relative '../classes/musicalbum'
+require_relative '../classes/genre'
```

Comment:
- [ ] I was unable to run these tests because the musicalbum file is not present in your codebase; perhaps you meant music.rb. 🙏 Kindly resolve it.

<img width="592" alt="Untitled" src="https://github.com/martinkarugaba/my-catalog/assets/103386183/7d882795-2144-40a1-90d5-f29f543df76f">